src/lightly_train/_commands/train_task.py

```diff
@@ -286,13 +281,15 @@ def _train_task_from_config(config: TrainTaskConfig) -> None:
                 max_steps=config.steps,
                 log_dict=train_log_dict,
             )
-            # TODO(Guarin, 07/25): Consider logging this under a "debug" prefix?
             for group in optimizer.param_groups:
-                train_log_dict[f"lr/{group['name']}"] = group["lr"]
-                train_log_dict[f"wd/{group['name']}"] = group["weight_decay"]
+                train_log_dict[f"learning_rate/{group['name']}"] = group["lr"]
+                train_log_dict[f"weight_decay/{group['name']}"] = group["weight_decay"]
             fabric.log_dict(train_log_dict, step=step)
             helpers.reset_metrics(train_result.log_dict)
 
+        if is_save_ckpt_step or is_last_step:
+            helpers.save_checkpoint(fabric=fabric, out_dir=out_dir, state=state)
```

Comment:
Moved this further up to save the checkpoint before validation in case there is an error during validation.